cvat-core/src/frames.ts

```diff
@@ -38,6 +38,7 @@ export class FrameData {
         stopFrame,
         decodeForward,
         deleted,
+        downloadOriginal,
```

Comment:
I would suggest to implement it via cvat-core global settings.

In ``cvat-core/src/config.ts`` define something like: ``chunkDataQuality: 'compressed'`` where ``compressed`` is a default value. 

When you change this setting via the interface, need to call:

```js
import { getCore } from 'cvat-core-wrapper';
getCore().config.chunkDataQuality = 'original' | 'compressed';
```

Much less of code changes expected.

Reply:
And to use this setting in cvat-core:

```
import config from './config';

if (config.chunkDataQuality ) { ... }
```

Reply:
Hi @bsekachev,
Is there an example of changing the cvat-core global settings via the UI? I'm trying to figure out how this would be done that's consistent with how it's currently done with other settings in player-settings.tsx. I can probably modify the global settings directly in mapDispatchToProps but I don't think that's a clean solution.

Reply:
Probably this settings should be added to the settings reducer to be able to save and restore it when updating a page (**as you already done it**). 

To update cvat-core settings, good enough option is to use React hooks when mount or update the checkbox component, I believe. Example:


```tsx
import { getCore } from 'cvat-core-wrapper';

function ChunkDataQualityInputComponent({ value }: { value: 'compressed' | 'original' }) {
  useEffect(() => {
      getCore().config.chunkDataQuality = value;
  }, [value]);
  
  return (<jsx code for checkbox>)
}

export default React.memo(ChunkDataQualityInputComponent);
```

and using like:

```
<ChunkDataQualityInputComponent value={value from store} />
```

Reply:
Hi @bsekachev
Thanks for the advice. Having a few issues implementing this:
1. Because the setting only gets set on checkbox render, any data downloaded before the settings modal is open uses the default value. Not too sure how to get around this, especially to render it before the initial chunk request is sent.
2. I can't seem to get config.chunkDataQuality to pass through to server-proxy.ts. Not sure if it's something I'm doing. I might be passing by copy somewhere instead of by reference?
Please see latest commit 7b784341b8792c98ef33be63e0d27241218c6c7b for my implementation.
Any guidance welcome.

Reply:
Hi @brendando-csiro 

Sorry for the delay, just a lof of work. I fixed your comments and now it looks working. 
But there is still one issue. The patch works well for images, but not for videos. 
Because originalChunkType and compressedChunkType are different for videos. And client tries to unzip a videofile.
If you have time to look at this, would be great, if not, I will do it later. 

<img width="272" alt="image" src="https://user-images.githubusercontent.com/40690378/219620977-afb616ab-273d-4cd2-ba2b-1c3ea507bde1.png">

Reply:
Thanks for that @bsekachev 
I can have a look at that issue later this week or next week.